internal/command/user_v2_invite.go

```diff
@@ -50,8 +51,10 @@ func (c *Commands) sendInviteCode(ctx context.Context, invite *CreateUserInvite,
 	if err != nil {
 		return nil, nil, err
 	}
-	if err := c.checkPermission(ctx, domain.PermissionUserWrite, wm.ResourceOwner, wm.AggregateID); err != nil {
-		return nil, nil, err
+	if wm.AggregateID != "" && wm.AggregateID != authz.GetCtxData(ctx).UserID {
```

Comment:
I'd reused the existing helper function, but am ok with directly checking the user here. (wm.AggregateID check is unneeded / redundant):
```suggestion
	if wm.AggregateID != authz.GetCtxData(ctx).UserID {
```

Please also add a test (at least for the Resend), where a user is allowed to send it's own code.